packages/graphql/src/schema/mutation.ts

```diff
@@ -450,13 +451,27 @@ export const Mutation = builder.mutationType({
         nullable: true,
         args: {
           elementInstanceId: t.arg.int({ required: true }),
+          elementId: t.arg.int({ required: true }),
           content: t.arg.string({ required: true }),
         },
         async resolve(_, args, ctx) {
           return ParticipantService.flagElement(args, ctx)
         },
       }),
 
+      rateElement: t.withAuth(asParticipant).field({
+        nullable: true,
+        type: ElementFeedback,
+        args: {
+          elementInstanceId: t.arg.int({ required: true }),
+          elementId: t.arg.int({ required: true }),
```

Comment:
why do we need both ids to rate it? is the elementId not inside the instance already?